brainy-ideas/readAloud-valence-jess/readme.md

```diff
@@ -0,0 +1,145 @@
+    - variables 3+ (1B only): participant measures on initial questionnaires (BMIS, etc.) {continuous}
+
+    **Dependent Variables**
+    - speed of reading aloud: syllables (required to read the text, not necessarily as produced) / second {continuous}
+    - percentage of disfluent syllables: number of disfluent syllables (repeated syllable, pre-syllable pause or interjection, and/or mispronounced syllable) / syllables required to read the text aloud {continuous}
+
+_Prediction 2_
+
+Sass et al. (2012) found an interesting pattern in valence priming: positive words are highly effective in priming other positive words, but negative words don't really "prime" anything (that is, RT on lexical decision for target words is similar for both related (negative) and unrelated (positive) targets following a negative prime).  More interesting yet, when they compared RTs for positive-prime>negative-target against negative-prime>positive-target, participants actually performed better in the **latter**.  If positive words activate a larger semantic network whereas compensatory mechanisms prevent such extensive network activation following exposure to negative words, one would expect a greater likelihood of disfluency at a positive>negative switch than a negative>positive switch.  Alternately, if shifting between valence contexts is akin to task-switching, the surprisal associated with the conflicting valence would be expected to impede performance in either direction, but particularly when hitting a valenced word contradictory to one's current mood state.
+* **A**: positive activation and negative compensation: fewer disfluencies in negative>positive switches than reverse
+* **B**: general surprisal: similar disfluency rates positive>negative and negative>positive
+* **C**: mood-incongruent surprisal: enhanced disfluency rates when hitting switch that conflicts with current mood state
+
+    **Independent Variable**
+    - variable 1: type of switch (pos>neg or neg>pos) {categorical}
+    - variables 2: participant measures on initial questionnaires (BMIS, etc.) {continuous}
+
+    **Dependent Variable**<br/>
+    percentage of disfluent syllables across three factors:
+
+    | pre-switch group| switch group | post-switch group|
+    |:-: | :-: | :-: |
+    | 7 - 6 - 5 - 4 - 3 | 2 - 1 - switch - 1 - 2 | 3 - 4 - 5 - 6 - 7 |
+
+    1. within the switch group (two words prior to the switch, the switch itself, and two words following the switch)
+    2. in the word group prior to the switch (the five words prior to the switch group)
+    3. in the post-switch word group (the five words that following the switch group)
+
+    *Note 1*: I feel that syllables form a logical 'unit' in which to measure disfluency.  However, as semantic integration is required to prepare articulation during reading aloud, I propose that we slice on the word level for analyzing disfluency around the switch.<br/>
```

Comment:
Things like this will be good to get Angela's input on :) No need to finalize the measures in brainbox though;